mobilecoind-dev-faucet/src/lib.rs

```diff
@@ -64,13 +64,19 @@ pub struct Config {
     /// Target Queue Depth. When the queue for a token id is less than this in
     /// depth, the worker attempts to make a split Tx to produce more TxOuts
     /// for the queue.
-    #[clap(long, default_value = "10000", env = "MC_TARGET_QUEUE_DEPTH")]
+    #[clap(long, default_value = "100000", env = "MC_TARGET_QUEUE_DEPTH")]
```

Comment:
Why did we up this default?

Reply:
because in the graphs we generated, we didn't really sustain the peak in a way that makes it clear what the peak is

historically slam used 100,000 txs -- I went down to lower numbers for ease of development and initial testing, and i was kind of hopefuly that 10 or 20 thousand would be enough to see the peak, but it looks like in a deployed network it's not enough